.github/workflows/inputs-remote-copy.yml

```diff
@@ -0,0 +1,90 @@
+name: Config Inputs Remote Copy
+run-name: "${{ inputs.remote-environment }} Config Input Copy to ${{ inputs.target }}"
+on:
+  workflow_dispatch:
+    inputs:
+      remote-environment:
+        type: choice
+        required: true
+        description: The Github Environment for the given remote
+        options:
+          - Gadi
+      source:
+        type: string
+        required: true
+        description: Remote absolute path to configuration input file/folder
+      target:
+        type: string
+        required: true
+        description: Remote absolute path for the copied configuration input file/folder
```

Comment:
We *could* (not saying this is a good idea) split the path into a root and a final calVer sub-dir, and provide a default for the latter to help users who aren't sure how to version inputs.

Reply:
I think in the first instance, we should start with explicit commands (as that is what users were originally doing (`cp` their work into the input dir)), and add helpful stuff incrementally